### PR TITLE
Update azumi.is-an.app.json

### PR DESCRIPTION
**Pull request by AquaQuokka**
I will use this domain to access my documentation for my Discord bot

### Link to Website

Link: 
https://azumi.is-an.app/docs
---

- [] Toggle this checkbox to re-run the stargazer checking

### CODE REVIEW

File: domains/azumi.is-an.app.json [hunk 1]
```diff
@@ -12,9 +12,9 @@
   },
   "nested": [
     {
-      "subdomain": "railway",
+      "subdomain": "docs",
       "record": {
-        "CNAME": "azumi.up.railway.app"
+        "CNAME": "azumi-development.github.io/docs"
```
Comment (tarampampam) [inline domains/azumi.is-an.app.json:17]: `azumi-development.github.io/docs` is not a domain name and cannot be used here

**Review by tarampampam — CHANGES_REQUESTED** [review]

File: domains/azumi.is-an.app.json (hunk 1 above)
Comment (AquaQuokka) [reply domains/azumi.is-an.app.json:17]: yeah i know, what record can i use to do this though?

Comment (tarampampam) [reply domains/azumi.is-an.app.json:17]: None. You may be confusing DNS and routing

Comment (tarampampam) [reply domains/azumi.is-an.app.json:17]: DNS is for resolving IP addresses from domain names. It does not know anything about, URLs, HTTP, HTTPS, and so on. Just domain name => IP address, not more

Comment (AquaQuokka) [reply domains/azumi.is-an.app.json:17]: welp that's fine
but how do i set up email stuff and that, because i kinda need to get that workin

Comment (AquaQuokka) [reply domains/azumi.is-an.app.json]: aaaaaaaaaaa it doesn't support it

**Review by AquaQuokka — COMMENTED** [review]
.

**Review by tarampampam — APPROVED** [review]

**Pull request merged**